### ISSUE TITLE
HINCRBYFLOAT changes hash value type to float

### ISSUE THREAD

**Issue by ze-phyr-us** [jamesls/fakeredis#90]
redis-py always returns hash values as bytes, even when they are valid floating point numbers. fakeredis is consistent with that when you set the value with HSET:

```
In [1]: import redis, fakeredis

In [2]: real = redis.StrictRedis()
In [3]: real.hset('h', 'x', 1.5)
Out[3]: 1
In [4]: real.hgetall('h')
Out[4]: {b'x': b'1.5'}

In [5]: fake = fakeredis.FakeStrictRedis()
In [6]: fake.hset('h', 'x', 1.5)
Out[6]: 1
In [7]: fake.hgetall('h')
Out[7]: {b'x': b'1.5'}
```

However, it breaks when you use HINCRBYFLOAT:

```
# Standard behavior
In [8]: real.hincrbyfloat('h2', 'x', 1.5)
Out[8]: 1.5
In [9]: real.hgetall('h2')
Out[9]: {b'x': b'1.5'}

# Incompatible behavior
In [10]: fake.hincrbyfloat('h2', 'x', 1.5)
Out[10]: 1.5
In [11]: fake.hgetall('h2')
Out[11]: {b'x': 1.5}
In [12]: fake.hget('h2', 'x')
Out[12]: 1.5
```

HINCRBY has the same problem, it changes the type to int:

```
In [13]: real.hincrby('h3', 'x', 15)
Out[13]: 15
In [14]: real.hgetall('h3')
Out[14]: {b'x': b'15'}

In [15]: fake.hincrby('h3', 'x', 15)
Out[15]: 15
In [16]: fake.hgetall('h3')
Out[16]: {b'x': 15}
In [17]: fake.hget('h3', 'x')
Out[17]: 15
```

**Comment by ze-phyr-us** [comment]
I started fixing this here: https://github.com/ze-phyr-us/fakeredis/commit/859e0e5edaf82e1e95ce57d8444ea1e0d233a4ee

It seems that incr, decr and friends have the same problem. I'm happy to fix them too and submit a PR later this week when I have more time.

**Comment by ikreymer** [comment]
I just ran into this issue as well with `hincrby`..  Are you still interesting in submitting this PR?

**Comment by bmerry** [comment]
This is fixed in [fakenewredis](https://github.com/ska-sa/fakenewsredis), a fork of fakeredis.

**Comment by bmerry** [comment]
Fixed in 0.10.1.
